app/src/main/java/org/mifos/selfserviceapp/ui/fragments/SavingAccountsDetailFragment.java

```diff
@@ -100,6 +102,7 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getArguments() != null) {
             savingsId = getArguments().getLong(Constants.SAVINGS_ID);
+            accountId = getArguments().getLong(Constants.ACCOUNT_ID);
```

Comment:
Revert back these changes because we already passing the SavingsAccount Id. don't need to pass thing twice. It's useless.